cves/CVE-2006-5752.yml

```diff
@@ -3,14 +3,15 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: CWE-79: Improper Neutralization of Input During Web Page Generation ('Cross-site Scripting')
```

Comment:
Please use just the CWE number (79) instead of words.

Reply:
```suggestion
CWE: 79
```

Reply:
Also, this is probably breaking your build (yml doesn't like colons here)